src/main/database/database.sql

```diff
@@ -0,0 +1,105 @@
+DROP TABLE IF EXISTS Action;
+CREATE TABLE Action (
+	id	 	INTEGER PRIMARY KEY AUTOINCREMENT,
+	name	TEXT
+);
+
+DROP TABLE IF EXISTS Log;
+CREATE TABLE Log (
+	id	 	INTEGER PRIMARY KEY AUTOINCREMENT,
+	actionId INTEGER REFERENCES Action(id),
+	companyId INTEGER REFERENCES Company(id),
+	timestamp	DATETIME,
+	extrainfo	TEXT
+);
+
+DROP TABLE IF EXISTS Company;
+CREATE TABLE Company (
+	id	 	INTEGER PRIMARY KEY AUTOINCREMENT,
+	name	TEXT,
+	contacts	TEXT NOT NULL, 
```

Comment:
Wouldn't it be better if there was a telephone number and an email and one of them had to exist?